assets/company-admin/components/CompanyAdminApp.jsx

```diff
@@ -149,8 +149,8 @@ class CompanyAdminAppComponent extends React.Component {
                                 </button>
                             </div>
 
-                            {this.props.sectionId === 'users' ? <label className='label label--big label--rounded'>{
-                                gettext('Users : {{total}}', {total : this.props.totalUsers})}</label> : ''}
+                            {this.props.sectionId === 'users' ? <label className='label label--big label--rounded me-2'>{
+                                gettext('UTILISATEURS : {{total}}', {total : this.props.totalUsers})}</label> : ''}
```

Comment:
this should be again `'Users'` in the gettext

Reply:
done